### DISCUSSION TITLE
No nativeless version anymore?

### DISCUSSION THREAD

**Discussion by Jenix-Park** [melanchall/drywetmidi#226] [Q&A] [answered]
Thanks for the new version!
By the way, 6.1.2 had nativeless versions, but there aren't any in 6.1.3.
No more nativeless?

**Comment by melanchall** [comment, marked as answer]
Hmm, but... it is there :)

https://www.nuget.org/packages/Melanchall.DryWetMidi.Nativeless

![image](https://user-images.githubusercontent.com/26361185/198236214-5159bd89-f82c-4e1c-b8d7-2968cf8090bc.png)

Do you have any problems with installing it?

**Reply by Jenix-Park** [reply]
Oh, there is!
Thanks.
The page I visited was [here](https://github.com/melanchall/drywetmidi/releases/tag/v6.1.3).

**Reply by melanchall** [reply]
That page is a common release notes for the new version of the library. But the library itself is shipped in two editions: full and nativeless. So every new release will lead to both editions updated :)
